test/utils/runCommand.test.ts

```diff
@@ -25,24 +53,25 @@ describe('commandRunner', () => {
       const context = workspaceManager.getContext(textDoc.uri);
       const settings = await context.documentSettings.get(textDoc.uri);
 
-      const commandRunner = new CommandRunner(
-        connection,
-        context,
-        settings
-      );
+      const commandRunner = new CommandRunner(connection, context, settings);
       try {
-        const proc = await commandRunner.runCommand(args[0], args.slice(1).join(' '));
+        const proc = await commandRunner.runCommand(
+          args[0],
+          args.slice(1).join(' '),
+          undefined,
+          undefined,
+          keepAnsi
+        );
         expect(proc.stdout).contains(stdout);
         expect(proc.stderr).contains(stderr);
-        }
-      catch (e) {
+      } catch (e) {
         if (!isWindows()) {
```

Comment:
The condition the test won't run on windows at which is shouldn't be the case. We want to run the test on windows with WSL2 enabled as it is in our supported matrix.